src/index.js

```diff
@@ -35,7 +35,8 @@ function format({
 
   try {
     const pretty = prettify(text, prettierOptions)
-    return eslintFix(pretty, eslintConfig)
+    const eslintFixed = eslintFix(pretty, eslintConfig)
+    return eslintFixed
```

Comment:
Made this a variable name just so I can log it easier. Thinking that it could be useful to expose better logging control to log what the text is (if you want a silly amount of logs for example).